File: Dotnet.Core/RomanNumeralConsole/RomanNumeralTest/RomanNumeralConverterShould.cs
using NUnit.Framework;
using RomanNumeralConsole;

namespace RomanNumeralTest
{
    public class RomanNumeralConverterShould
    {
        [Test]
        public void Convert_arabic_numbers_into_their_respective_roman_numeral()
        {
            const string assertedRoman = "I";
            var roman = RomanNumeralConverter.RomanFor(1);
            Assert.AreEqual(roman, assertedRoman, $"Arabic conversion should be {assertedRoman} instead of {roman}");
        }
    }
}

File: Java/Start/test/RomanNumeralConverterShould.java
import junitparams.JUnitParamsRunner;
import junitparams.Parameters;
import org.junit.Test;
import org.junit.runner.RunWith;

import static org.hamcrest.core.Is.is;
import static org.junit.Assert.assertThat;

@RunWith(JUnitParamsRunner.class)
public class RomanNumeralConverterShould {

    @Test
    public void convert_arabic_numbers_into_their_respective_roman_numeral() {
        int arabic = 1;
        assertThat(RomanNumeralConverter.romanFor(arabic), is("I"));
    }
}